docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10-alpine
+    ports:
+      - "5432:5432"
```

Comment:
I don't know how docker networking works, but doesn't that open the port to everything instead of just the required other containers?

Reply:
at the moment it's there for my convienient so i can pg in to the box as well to inspect it; 

that said, it opens it to the host machine, if the host machine port forwards that to the public, yes, everyone can access it.